<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YxWn-Gallery</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
	border-image: url(C:/Users/Admin/Desktop/Coding  Tools/Qt/YxWn_Gallery/resources/ShiroNana.jpeg) 0 0 0 0 stretch stretch;
/*
	background-image: url(C:/Users/Admin/Downloads/qttestfolder/ShiroNana.jpeg) ;
	background-position: center;
	background-size: inherit;
*/
}
</string>
   </property>
   <widget class="QLabel" name="img">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1216</width>
      <height>491</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Pictures/Twitter/25f7218c010fdff4c6e0a4d838cc820c.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QVideoWidget" name="vid" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1216</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="contActionPanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>1216</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_file">
     <widget class="QLabel" name="lblPath">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Path:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblFilePath">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>100</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File Path:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSelectFolder">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblFileName">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>130</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGenerate">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>60</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Roll</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_player">
     <widget class="QSlider" name="progressBar">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>1071</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPlay">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>80</y>
        <width>52</width>
        <height>52</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:25px;
 border-color: red;</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSkip">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>80</y>
        <width>52</width>
        <height>52</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:25px;
 border-color: red;</string>
      </property>
      <property name="text">
       <string>Skip
30</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRewind">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>52</width>
        <height>52</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: grey;
 border-style: solid;
 border-width:1px;
 border-radius:25px;
 border-color: red;</string>
      </property>
      <property name="text">
       <string>Rewind
10</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrentTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblRemainingTime">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>contActionPanel</zorder>
   <zorder>img</zorder>
   <zorder>vid</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
